packages/data-provider/README.md

```diff
@@ -0,0 +1,173 @@
+# Lyric (Data Provider)
+
+Package published on npm as https://www.npmjs.com/@overture-stack/lyric
+
+This package is part of Lyric mono repo, for more details go to [main page](../../README.md)
+
+## Install
+
+```
+npm i @overture-stack/lyric
```

Comment:
pnpm

Reply:
we can recommend pnpm for dev work, but however people want to import our package should be up to them